internal/scorecard/alpha/scorecard.go

```diff
@@ -175,10 +206,10 @@ func (r PodTestRunner) RunTest(ctx context.Context, test Test) (*v1alpha3.TestSt
 // RunTest executes a single test
 func (r FakeTestRunner) RunTest(ctx context.Context, test Test) (result *v1alpha3.TestStatus, err error) {
 	select {
+	case <-time.After(r.Sleep):
+		return r.TestStatus, r.Error
 	case <-ctx.Done():
 		return nil, ctx.Err()
-	default:
-		return r.TestStatus, r.Error
```

Comment:
should we not keep the default?

Reply:
No because default in a select statement is used when none of the channels have available items. If we left default in, it would _always_ win.